src/Models/ExampleModel.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace App\Models;
+
+
+use PDO;
+
+class ExampleModel
```

Comment:
maybe we can change this so that it's a realistic example e.g. `PigModel`, `TweetModel`, or something like that?

the reason being that students are often confused by MVC and i like to hammer in at every opportunity that we have 1 MODEL PER DB ENTITY and 1 CONTROLLER PER ROUTE

i find that they often think there's 1 MODEL PER CONTROLLER and they get everything confused from there - so i would rather give them something more realistic and then they can delete it within their own apps. does that sound like a good idea?

Reply:
So my plan would be to actually remove this from the skeleton - I'm torn on whether or not we should give them the PDO factory by default, but I just added this to show you guys how the autowiring works.